staff/gustavo-rojas/course-1-new-version/task1/about.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>NorCal Time Trials Series</title>
+  <link rel="stylesheet" href="styles/nctss_1.css">
+</head>
+
+<body>
+  <div class=logo-container>
+    <img class="logo" src="images/logo.png" alt="The Logo of NorCal Time Trial Series">
+  </div>
+  <div class=nameofthevent-container>
+    <p class="nameoftheevent"> NorCal Time Trial Series</p>
+  </div>
+  <div class="horizontal-image-container">

```

Comment:
What is the meaning of this div, it is iin all pages but i cannot see the image.

Reply:
Done .

Reply:
Removed the division .